cp3pt0-deployment/common/utils.sh

```diff
@@ -387,12 +387,17 @@ function is_sub_exist() {
 }
 
 function check_cert_manager(){
-    csv_count=`$OC get csv |grep "cert-manager"|wc -l`
+    csv_count=$(${OC} get csv -A | grep "cert-manager"| wc -l | tr -d " ")
     if [[ $csv_count == 0 ]]; then
         error "Missing a cert-manager"
     fi
+    # if installed in all namespace mode or alongside cp2 cert manager, 
+    # csv_count will be >1, need to check for multiple deployments
     if [[ $csv_count > 1 ]]; then
-        error "Multiple cert-manager csv found. Only one should be installed per cluster"
+        webhook_deployments=$(${OC} get deploy -A --no-headers --ignore-not-found | grep "cert-manager-webhook" -c)
+        if [[ $webhook_deployments != "1" ]]; then
+            error "Multiple cert-managers found. Only one should be installed per cluster"
+        fi
```

Comment:
@bluzarraga 
> The changes to the utils.sh helper script also seem relevant to recently opened cpd-quality issue 9998

I believe it will impact `olm-utils` functionality that they only have tenant permission to get CSV, `-A` will require a cluster permission to do so.